docs/strategy-customization.md

```diff
@@ -451,6 +452,17 @@ if self.dp:
     The order book is not part of the historic data which means backtesting and hyperopt will not work if this
     method is used.
 
+#### *ticker(pair)*
+
+``` python
+if self.dp:
+    if self.dp.runmode.value in ('live', 'dry_run'):
+        ticker = self.dp.ticker(metadata['pair'])
+        dataframe['last_price'] = ticker['last']
```

Comment:
I think we should note here that fields can depend on the exchange (not all exchanges return vwap for example) - and FTX does not always return last field - so it can be None ...